community/modules/remote-desktop/chrome-remote-desktop/variables.tf

```diff
@@ -0,0 +1,260 @@
+      network_tier           = string
+    })),
+    ipv6_access_config = list(object({
+      public_ptr_domain_name = string,
+      network_tier           = string
+    })),
+    alias_ip_range = list(object({
+      ip_cidr_range         = string,
+      subnetwork_range_name = string
+    }))
+  }))
+  default = []
+}
+
+variable "metadata" {
+  description = "Metadata, provided as a map"
+  type        = map(string)
+  default     = {}
+}
+
+variable "startup_script" {
+  description = "Startup script used on the instance"
+  type        = string
+  default     = null
+}
+
+variable "guest_accelerator" {
+  description = "List of the type and count of accelerator cards attached to the instance. Requires virtual workstation accelerator if Nvidia Grid Drivers are required"
+  type = list(object({
+    type  = string,
+    count = number
+  }))
+  default = [{
+    type  = "nvidia-tesla-t4-vws"
+    count = 1
+  }]
+}
+
+variable "threads_per_core" {
+  description = "Sets the number of threads per physical core"
+  type        = number
+  default     = 2
+}
+
+variable "on_host_maintenance" {
+  description = "Describes maintenance behavior for the instance. If left blank this will default to `MIGRATE` except for when `placement_policy`, spot provisioning, or GPUs require it to be `TERMINATE`"
+  type        = string
+  default     = "TERMINATE"
+}
+
+variable "bandwidth_tier" {
+  description = <<EOT
+  Tier 1 bandwidth increases the maximum egress bandwidth for VMs.
+  Using the `tier_1_enabled` setting will enable both gVNIC and TIER_1 higher bandwidth networking.
+  Using the `gvnic_enabled` setting will only enable gVNIC and will not enable TIER_1.
+  Note that TIER_1 only works with specific machine families & shapes and must be using an image th
+at supports gVNIC. See [official docs](https://cloud.google.com/compute/docs/networking/configure-v
+m-with-high-bandwidth-configuration) for more details.
+  EOT
+  type        = string
+  default     = "not_enabled"
+}
+
+variable "spot" {
+  description = "Provision VMs using discounted Spot pricing, allowing for preemption"
+  type        = bool
+  default     = false
+}
+
+variable "tags" {
+  description = "Network tags, provided as a list"
+  type        = list(string)
+  default     = []
+}
+
+variable "enable_oslogin" {
+  description = "Enable or Disable OS Login with \"ENABLE\" or \"DISABLE\". Set to \"INHERIT\" to inherit project OS Login setting."
+  type        = string
+  default     = "ENABLE"
+}
+
+variable "install_nvidia_driver" {
+  description = "Installs the nvidia driver (true/false). For details, see https://cloud.google.com/compute/docs/gpus/install-drivers-gpu"
+  type        = bool
+}
+
+variable "configure_chrome_remote_desktop" {
+  description = "This will trigger the installation of chrome remote desktop (true/false). For details, see https://cloud.google.com/architecture/chrome-desktop-remote-on-compute-engine"
+  type        = bool
+  default     = true
```

Comment:
It seems odd to me to provide this option on the Chrome Remote Desktop module. What is a Chrome Remote Desktop without Chrome Remote Desktop?

Reply:
I agree, the original thought was to re-use this module as a base to optionally install CRD, Guacamole, Windows/Linux etc using user variables, but that could come later if we can preview CRD only first.

I'll update `main.tf` to run the ansible runner and remove this option from `variables.tf`